mcs/kubernetes_cluster_client.go

```diff
@@ -59,34 +59,34 @@ func flattenNodes(nodes []*Node) nodesFlatSchema {
 	return flatSchema
 }
 
-//NodeGroupClusterPatchOpts ...
+// NodeGroupClusterPatchOpts ...
 type NodeGroupClusterPatchOpts []NodeGroupPatchParams
 
-//NodeGroupPatchParams ...
+// NodeGroupPatchParams ...
```

Comment:
When do you plan to fill docstrings?

Reply:
it's planned in future PRs, this one is just to add the linters